README.md

```diff
@@ -366,6 +366,15 @@ If you have a different question, please ask using [Discussions](https://github.
   RSS Feed plugin works with these correctly set up fields: `title`, `url`, `description` and `author`.
   Make sure to fill them in an appropriate way and try again.
 
+5. **Q:** My site doesn't work when I enable `related_blog_posts`. Why?
```

Comment:
```suggestion
5. **Q:** My site doesn't work when I enable `related_blog_posts`. Why? <br>
```
if you don't add `<br>`, github shows `Q` and `A` parts in one line / same paragraph, which is harder to read.